package.json

```diff
@@ -100,10 +100,10 @@
   "peerDependencies": {
     "@storybook/svelte": "^7.0.0",
     "@storybook/theming": "^7.0.0",
-    "@sveltejs/vite-plugin-svelte": "^2.0.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.0",
     "svelte": "^4.0.0",
     "svelte-loader": "^3.1.2",
-    "vite": "^4.0.0"
+    "vite": "^5.0.0"
```

Comment:
```suggestion
    "vite": "^4.0.0 || ^5.0.0"
```

Reply:
So my only concern here is that this could result in some things breaking for some users. I'm not 💯 sure about this codebase and whether any of the breaking changes apply though, so that may not be the case.

https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#breaking-changes